src/brucke.erl

```diff
@@ -0,0 +1,37 @@
+%%%
+%%%   Copyright (c) 2018 Klarna Bank AB (publ)
+%%%
+%%%   Licensed under the Apache License, Version 2.0 (the "License");
+%%%   you may not use this file except in compliance with the License.
+%%%   You may obtain a copy of the License at
+%%%
+%%%       http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%%   Unless required by applicable law or agreed to in writing, software
+%%%   distributed under the License is distributed on an "AS IS" BASIS,
+%%%   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%%%   See the License for the specific language governing permissions and
+%%%   limitations under the License.
+%%%
+
+%% @doc This is the main module for config validation escript
+-module(brucke).
+
+-export([main/1]).
+
+main([File]) ->
+  case os:getenv("BRUCKE_FILTER_MODULE_BEAM_DIRS") of
```

Comment:
IMO both parameters should be passed on the command-line.

Reply:
fixed